mule-artifact-it/src/test/java/integrationTests/mojo/AbstractMojoTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package integrationTests.mojo;
+
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+
+import java.io.File;
+
+public interface AbstractMojoTest {
```

Comment:
if it is an interface it should not be name Abstract 
How about settingsConfigurator? 
Or make AbstractMuleMojo an abstract Class

Reply:
Ok!